### PR TITLE
Updated 3D center rules.

### PR DESCRIPTION
**Pull request by jbreitbart**
These are the rules used by the team running the website.

### CODE REVIEW

**Comment by jbreitbart** [comment]
Any comment on what I should do to fix this?

**Comment by jsha** [comment]
To fix the tests, just remove:

```
+ <target host=".3dcenter.org" />
```

It's not needed.

File: src/chrome/content/rules/3DCenter.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="3DCenter.org (cacert)" platform="cacert">
+	<target host="www.3dcenter.org" />
+	<target host="m.3dcenter.org" />
```
Comment (julianladisch) [inline src/chrome/content/rules/3DCenter.xml]: add
    <target host="3dcenter.org" />
    <rule from="^http://3dcenter\.org/" to="https://www.3dcenter.org/"/>

File: src/chrome/content/rules/3DCenter.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	<rule from="^http://www\.3dcenter\.org/" to="https://www.3dcenter.org/"/>
+	<rule from="^http://m\.3dcenter\.org/" to="https://m.3dcenter.org/"/>
```
Comment (julianladisch) [inline src/chrome/content/rules/3DCenter.xml]: It is more easy to understand if you replace the two rules for www.3dcenter.org and m.3dcenter.org by `<rule from="^http:" to="https:" />`.  There is a new [Ruleset Style Guide](https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md).

File: src/chrome/content/rules/3DCenterForum.xml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="3DCenter.org Forum">
+	<target host="www.forum-3dcenter.org" />
```
Comment (julianladisch) [inline src/chrome/content/rules/3DCenterForum.xml]: Add
`<target host="forum-3dcenter.org" />`
`<rule from="^http://forum-3dcenter\.org/" to="https://www.forum-3dcenter.org/"/>`
`<rule from="^http:" to="https:"/>`
and remove the existing www.forum-3dcenter.org rule.

**Comment by julianladisch** [comment]
The checker complains about the cacert certificate, but the rule is tagged by platform="cacert". This is not an issue of the ruleset.  The checker should use the cacert root certificates when the ruleset is tagged with platform="cacert".

**Comment by semenko** [comment]
Closed since this has some outstanding work and hasn't been touched in a while. Feel free to re-open if you'd like this merged.

**Pull request closed without merging**